cpp/src/arrow/util/bit_block_counter.h

```diff
@@ -76,6 +76,9 @@ struct BitBlockCount {
   bool AllSet() const { return this->length == this->popcount; }
 };
 
+/// \brief Return the word step size of each bit block counter run
+int64_t BitBlockCounterWordSize();
```

Comment:
This should probably be a constant rather than a function.

Reply:
Thanks, done